source/base/subscriptor.cc

```diff
@@ -62,39 +62,57 @@ Subscriptor::Subscriptor (const Subscriptor &)
 {}
 
 
+
+#ifdef DEAL_II_WITH_CXX11
+Subscriptor::Subscriptor (Subscriptor &&subscriptor)
+  :
+  counter(0),
+  object_info (subscriptor.object_info)
+{
+  subscriptor.check_no_subscribers();
+  subscriptor.counter = 0;
+  subscriptor.counter_map.clear();
```

Comment:
These last lines are not wrong, but unnecessary.  We assume that the two variables are empty, and check so in the line above, so yes lines should be no ops.